pytest_reportportal/service.py

```diff
@@ -41,8 +40,7 @@ def init_service(self, endpoint, project, uuid, log_batch_size,
             self.ignore_errors = ignore_errors
             self.ignored_tags = ignored_tags
             logging.debug(
-                msg="ReportPortal - Init service: "
-                    "endpoint={0}, project={1}, uuid={2}".
+                msg='ReportPortal - Init service: endpoint={0}, project={1}, uuid={2}'.
```

Comment:
Please limit line length up to 79 chars (inclusive).

Reply:
Fixed